src/commands/generate/__tests__/resources/services/outOfOrderPathService.ts

```diff
@@ -0,0 +1,30 @@
+import { IHttpApiBridge, MediaType } from "conjure-client";
+
+export interface IOutOfOrderPathService {
+    foo(param1: string, param2: string): Promise<void>;
+}
+
+export class OutOfOrderPathService {
+    constructor(private bridge: IHttpApiBridge) {
+    }
+
+    public foo(param1: string, param2: string): Promise<void> {
+        return this.bridge.callEndpoint<void>({
+            data: undefined,
+            endpointName: "foo",
+            endpointPath: "/{param2}/{param1}",
+            headers: {
+            },
+            method: "GET",
+            pathArguments: [
+                param2,
+
```

Comment:
this bonus newline is bothering me (it looks to be present in all the other generated testcases too) but couldn't figure out where it's coming from